notebooks/concepts/Specviz from splot concept notebook.ipynb

```diff
@@ -0,0 +1,3636 @@
+   "metadata": {},
+   "source": [
+    "    Internally this should do the same as loading the `Spectrum1D` - i.e. first convert it to `Spectrum1D` using `read` and then load it into glue/jdaviz. That way the user can know exactly what is happening when they do it with a GUI button or at the command line."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2. In the UI, the above should be possible using the \"open\" or \"import data\" button - the intention would be for it to all behave just as described above, but via a file browser (or command-line file path) instead of via a notebook method call."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Multiple spectra in one `specviz` "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Because glupyter/glue support multiple data being loaded at the same time, it should also be possible to load data even if it's not all getting displayed initially.\n",
+    "\n",
+    "1.  In the notebook this is just a different method that otherwise behaves the same as `load_data`, just doesn't replace what's being shown - E.g.,"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "specviz.add_data('some_other_spectrum.fits') #<- this would do exactly the same as `load_data` but just not show it\n",
+    "# OR \n",
+    "specviz.add_data('some_other_spectrum.fits', show=True)  #<- this would automatically add the flux as a new layer in the UI"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2. In the UI, if the user does \"import data\" a second (or additional) time, `add_data` as described above can be used instead of `load_data`, as the user can always simply go into the viewer and change what's being shown via check boxes/drag-and-drop."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Uncertainties/mask"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If the `Spectrum1D` object loaded contains an uncertainty and/or mask attribute, these should be loaded ointo glue when `load_data` is called.  However, by default only the spectrum itself should be *shown* at that time. The user can show it later using standard `jdaviz` interactions if desired. We *may* wish some syntactic convenience for the notebook as well - e.g.:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "specviz.show_uncertainty()\n",
+    "specviz.show_mask()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Accessing the spectrum\n",
+    "\n",
+    "(only relevant for notebook/developer usage) - When a single spectrum is loaded using `load_data`, it should be easy to get that spectrum back again.  E.g.,"
+   ]
```

Comment:
Ah, I see what you mean.  Here's why I think the accessor is needed:

1. If the user hits the "import" button in the notebook instead of using `load_data`, they don't have a copy of the spectrum to work with anywhere except in the viz tool itself, so need to be able to access it.
2. I think glue has to make a copy of the spectrum.  If so, then any modifications that might be made to the spectrum in the viz tool won't be available to the user without getting the viz tool's copy of the spectrum
3. If the user loads multiple spectra, the "current spectrum" might change depending on what the user does in the UI (i.e. they might hide the first one).  So this is really a way to say "apply some code to the spectrum I'm looking at right now in the viz tool regardless of where I got it from".

All of those are negotiable though...

Reply:
(Although now that I read on this talks to the next comment, which I think makes the case a bit more clear.)